src/V3Config.cpp

```diff
@@ -145,9 +146,311 @@ class V3ConfigIgnores {
 
 V3ConfigIgnores V3ConfigIgnores::s_singleton;
 
+//######################################################################
+// variable attributes (public, clock, isolate_assignments, ...)
+
+class V3ConfigVarAttr {
+    struct AttrEntry {
+        AstAttrType type;
+        FileLine*   flp; // currently don't use it, but may be needed for error reporting later
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [struct member 'AttrEntry::flp' is never used.](https://app.codacy.com/manual/wsnyder/verilator/pullRequest?prid=4742436)